File: cpp/sprint1_nonfinals/j.cpp
#include <iostream>
#include <vector>

using namespace std;

vector <int> Factorize(int number) {
	// Здесь реализация вашего решения

}

void print(const vector <int> &result) {
	for (int x : result)
		cout << x << " ";
	cout << endl;
}

int main() {
	int number;
	cin >> number;
	print(Factorize(number));
}

File: swift/sprint1_nonfinals/j.swift
func readInt() -> Int {
    return Int(readLine()!)!
}

func print(_ result: [Int]) {
    print(result.compactMap { String($0)}.joined(separator: " "))
}

func factorize(_ number: Int) ->[Int] {
    // Здесь реализация вашего решения
}

func main() {
    let number = readInt()
    print(factorize(number))
}

main()
